### PR TITLE
updates to trim packages and update versions

### PR DESCRIPTION
**Pull request by stargaser**
Update conda environment.

* python=3.8
* pin pyvo to the latest commit today, pending a v1.2 release
* remove unused packages

### CODE REVIEW

**Comment by stargaser** [comment]
My first attempt at pruning unused packages does not have much effect. `astroquery` depends on `requests`, and `aplpy` brings in `scikit-image` which needs `scipy` and many other packages.

**Comment by stargaser** [comment]
We'll want to test on Windows (Windows 10? Windows11?) and also on M1 MacBook.

File: environment.yml [hunk 1]
```diff
@@ -1,17 +1,15 @@
 name: navo-env
 channels:
-  - conda-forge
+  - astropy
 dependencies:
-  - python=3.7
+  - python=3.8
   - numpy
   - astropy>=4.1
```
Comment (tomdonaldson) [inline environment.yml:7]: I'm wondering a couple things about the astropy version, though not suggesting any change at this point:

- Currently this gives me v4.3.1, so we could up the minimum version to 4.3.1 as well.
- Version 5.0 may be released relatively soon.  I suspect we'll be fine with that, but we should definitely retest if/when that happens.

**Review by tomdonaldson — APPROVED** [review]
I think this looks good.  I added a comment about maybe being careful with a possible upcoming major release of `astropy`, but I don't have any concerns with this as written.

File: environment.yml (hunk 1 above)
Comment (stargaser) [reply environment.yml:7]: thanks TomD, we can watch the astropy version and re-test.

**Pull request merged**